r-package/paper2020/paper2020.Rmd

```diff
@@ -68,22 +74,27 @@ The package includes sample datasets for the cities of São Paulo and Porto Aleg
 
 ### Building a routable transport network
 
-To build a routable transport network with r5r and load it in memory, the user need to run a command line pointing to the location on disk where the input data is stored. In the examples of this paper, we will be using the provided Porto Alegre dataset.
+To build a routable transport network with r5r and load it into memory, the user needs to call `setup_r5` with the path to the directory where OSM and GTFS data are stored. In the examples herein, we use the provided Porto Alegre dataset.
 
+<!-- We're getting an error about OSM data here: Continuing to load but ignoring generalized costs due to exception: java.lang.RuntimeException: All ways are expected to have generalized cost tags. Missing: slope_1:forward
+Digging through the R5 code though, it looks like this is harmless and is related to some Los Angeles-specific development work done at Conveyal -->
 ```{r, message = FALSE}
 # points to directory with data
 data_path <- system.file("extdata/poa", package = "r5r")
 
 r5r_core <- setup_r5(data_path, verbose = FALSE)
 ```
 
-The function uses the `.pbf` and the `GTFS.zip` files in the directory pointed by `data_path` to create a multimodal transport network used for routing by R<sup>5</sup>. Multiple GTFS feeds can be used simultaneously, in which case R<sup>5</sup> will automatically merge all files into a single public transport network. The resulting `network.dat` as well as some other files used by R<sup>5</sup> are saved inside the supplied directory for later reuse.
+The function uses the `.pbf` and the `GTFS.zip` files in the directory pointed by `data_path` to create a multimodal transport network used for routing by R<sup>5</sup>. If multiple GTFS files are present, R<sup>5</sup> will merge them into a single transport network. The resulting `network.dat` as well as some other files used by R<sup>5</sup> are saved inside the supplied directory for later reuse.
 
 ### Calculating a travel time matrix
 
-The `travel_time_matrix()` function takes, as inputs, the spatial location of origins/destinations (either as a spatial `sf POINT` object, or as a `data.frame` containing the columns `id`, `lon` and `lat`) and a few travel parameters such as *maximum trip duration* or *walking distance*. It outputs travel time estimates for each origin-destination pair at a set `departure_datetime`.
+The `travel_time_matrix()` function takes, as inputs, the spatial location of origins/destinations (either as a spatial `sf POINT` object, or as a `data.frame` containing the columns `id`, `lon` and `lat`) and a few travel parameters such as *maximum trip duration*, or *walking distance*. It outputs travel time estimates for each origin-destination pair at a set `departure_datetime`.
+
+<!-- It might be worthwhile to consider changing the defaults to use a longer time window, and to return more percentiles, so users don't have to set parameters to see variation -MWC -->
```

Comment:
I belive most users are not yet familiar the idea of using a `time_window` , so they general user will likely use the function assuming the results refer to the exact departure time of input. I'm afraid that if we change the default `time_window` to 30 min or 1 hour, most users ( who don't read the documentation) will have a wrong interpretation of the output.  If we leave the `time_window` as it is now, the result is more intuitive for general users but we still allows for advanced users to tweak the `time_window`  parameter and fully benefit from the function capability.